### DISCUSSION TITLE
Typo in Disclosure docs

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#568] [General] [answered]
<img width="687" alt="Screen Shot 2020-07-10 at 4 32 05 PM" src="https://user-images.githubusercontent.com/67388401/87210603-0bc79400-c2cb-11ea-9def-f2b9dffc2912.png">
you are welcome ;)

**Comment by aayushpi** [comment]
Thanks Sang, can you help us by sharing where this typo is?

**Reply by sang-yeo** [reply]
Yup, I found it [here](https://paste.twilio.design/components/disclosure#controlling-initial-state)   **the intial state** should be **initial state**

**Comment by SiTaggart** [comment, marked as answer]
Pull request opened. Thanks for reporting it. https://github.com/twilio-labs/paste/pull/569
